content/de/verein/event/fireshonks-tracking-2022/index.md

```diff
@@ -7,7 +7,7 @@
         "online": "https://streaming.media.ccc.de/jev22/fireshonks"
     },
     "featured_image": "tracking-in-apps-ist-das-legal.png",
-    "license": "cc-by-40",
```

Comment:
Why remove the license?

Reply:
The video and the presentation are under different licenses:

https://github.com/datenanfragen/website/pull/1017/files#diff-100044f356f9a581c5f1530f0243fa231ea863184534ed4e49e20f9df3dea212R327

Specifying a single license there is just confusing.